src/components/icons/linkIcon.ts

```diff
@@ -0,0 +1,13 @@
+import {svg} from 'lit';
+import {color} from '../css/colors';
+
+export const linkIcon = svg`<svg width="16" height="17" viewBox="0 0 16 17" fill="none" xmlns="http://www.w3.org/2000/svg" class="w-5 h-5">
```

Comment:
Square?

Reply:
Moved to https://github.com/getAlby/bitcoin-connect/issues/52